src/tracker.rs

```diff
@@ -64,22 +65,55 @@ impl Tracker {
         }
     }
 
-    /// The `namespace` of this [Tracker]
+    /// The `namespace` of this Tracker
```

Comment:
The method name should imply its purpose - although I appreciate it might be customary to provide trivial comments on methods.